server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/GetMailboxes.feature

```diff
@@ -242,3 +242,10 @@ Feature: GetMailboxes method
     Then the mailbox "shared2" has 0 messages
     And the mailbox "shared2" has 0 unseen messages
 
+  Scenario: User can share sub-mailbox without sharing its parent
+    Given "alice@domain.tld" has a mailbox "mailbox1"
+    And "alice@domain.tld" has a mailbox "mailbox1.shared"
```

Comment:
"alice@domain.tld" has a mailbox "mailbox1.shared"

Should be enough.

Reply:
I prefer what she did, it make it clear mailbox1 is a selectable mailbox